cxplat/inc/cxplat.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+#pragma once
+
+#include "cxplat_fault_injection.h"
+#include "cxplat_memory.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
```

Comment:
To simplify this stuff, we can define `EXTERN_C` (or `EXTERN_C_START`) and `EXTERN_C_END` in `cxplat_common.h` for this. It would reduce the duplication and line count a bit. It would also hide the `{` and `}` to eliminate any visible/explicit need to tab things over.

Reply:
Filed #82